source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,321 @@
+destination deployments to ensure the safety of objects during transmission.
+
+MinIO does *not* support replicating client-side encrypted objects 
+(SSE-C).
+
+Replication of Locked Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports replicating objects held under 
+:ref:`WORM Locking <minio-object-locking>`. Both replication buckets *must* have
+object locking enabled for MinIO to replicate the locked object. For
+active-active configuration, MinIO recommends using the *same* 
+retention rules on both buckets to ensure consistent behavior across
+sites.
+
+You must enable object locking during bucket creation as per S3 behavior. 
+You can then configure object retention rules at any time.
+Object locking requires :ref:`versioning <minio-bucket-versioning>` and
+enables the feature implicitly.
+
+Procedure
+---------
+
+1) Configure User Accounts and Policies for Replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following examples use ``Alpha`` and ``Baker`` as placeholder :ref:`aliases
+<alias>` for each MinIO cluster. You should replace these values with the
+appropriate aliases for the MinIO deployments on which you are configuring
+bucket replication. These examples assume that the specified aliases have the
+necessary permissions for creating policies and users on both deployments. See
+:ref:`minio-users` and :ref:`MinIO Policy Based Access Control <minio-policy>`
+for more complete documentation on MinIO users and policies respectively.
+
+The following code creates the user and policies necessary for
+resynchronization of data from ``Alpha`` to ``Baker``. Replace the password
+``LongRandomSecretKey`` with a long, random, and secure secret key as per
+your organizations best practices for password generation.
+
+.. code-block:: shell
+   :class: copyable
+
+   wget -O - https://docs.min.io/minio/baremetal/examples/ReplicationAdminPolicy.json | \
+   mc admin policy add Alpha ReplicationAdminPolicy /dev/stdin
+   mc admin user add Alpha alphaReplicationAdmin LongRandomSecretKey
+   mc admin policy set Alpha ReplicationAdminPolicy user=alphaReplicationAdmin
+
+   wget -O - https://docs.min.io/minio/baremetal/examples/ReplicationRemoteUserPolicy.json | \
+   mc admin policy add Baker ReplicationRemoteUserPolicy /dev/stdin
+   mc admin user add Baker bakerReplicationRemoteUser LongRandomSecretKey
+   mc admin policy set Baker ReplicationRemoteUserPolicy user=bakerReplicationRemoteUser
+
+You can skip this step if
+both deployments already have users with the necessary :ref:`permissions
+<minio-bucket-replication-serverside-resynchronize-permissions>`.
+
+2) Configure ``mc`` Replication Admin Access
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the :mc-cmd:`mc alias set` command to add a replication-specific alias for
+the ``Alpha`` deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc alias set AlphaReplication HOSTNAME AlphaReplicationAdmin LongRandomSecretKey
+
```

Comment:
shouldn't this section be adding the remote target and adding replication rule?

Reply:
Agree, however since Baker went down - it would help user to check if the credentials and remote endpoint are still the same or need to be re-configured.